### ISSUE TITLE
Depends:   R (>= 3.6.0) is needn't

### ISSUE THREAD

**Issue by r-stata** [konradsemsch/ggrapid#4]
I think `Depends:   R (>= 3.6.0)` is needn't, `Depends:   R (>= 3.0.0)` is enough.

**Comment by konradsemsch** [comment]
Yeah, you're probably right over here. Have you tested it along with other dependencies?
